src/platforms/ios/iosiaphandler.h

```diff
@@ -23,7 +23,7 @@ class IOSIAPHandler final : public PurchaseIAPHandler {
 
  protected:
   void nativeStartSubscription(ProductsHandler::Product* product) override;
-  void nativeRestoreSubscription();
+  void nativeRestoreSubscription() override;
```

Comment:
How is this change related to the chekedInt change?

Reply:
No need for a separate PR, but I have the same question as above https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4979#discussion_r1029423504